docs/docs/themes.md

```diff
@@ -0,0 +1,103 @@
+---
+id: themes
+title: Themes
+sidebar_label: Themes
+---
+
+## oh-my-posh themes
+
+### [Agnoster]
+
+[![Agnoster](/img/themes/agnoster.png)][Agnoster]
+
+### [AgnosterPlus]
+
+[![AgnosterPlus](/img/themes/agnosterplus.png)][AgnosterPlus]
+
+### [Aliens]
+
+[![Aliens](/img/themes/aliens.png)][Aliens]
+
+### [Avit]
+
+[![Avit](/img/themes/avit.png)][Avit]
+
+### [DarkBlood]
+
+[![DarkBlood](/img/themes/darkblood.png)][DarkBlood]
+
+### [Emodipt]
+
+[![Emodipt](/img/themes/emodipt.png)][Emodipt]
+
+### [Fish]
+
+[![Fish](/img/themes/fish.png)][Fish]
+
+### [Honukai]
+
+[![Honukai](/img/themes/honukai.png)][Honukai]
+
+### [JanDeDobbeleer]
+
+[![JanDeDobbeleer](/img/themes/jandedobbeleer.png)][JanDeDobbeleer]
+
+### [Lambda]
+
+[![Lambda](/img/themes/lambda.png)][Lambda]
+
+### [Material]
+
+[![Material](/img/themes/material.png)][Material]
+
+### [ParaRussel]
+
+[![ParaRussel](/img/themes/pararussel.png)][ParaRussel]
+
+### [Powerlevel10k_Classic]
+
+[![Powerlevel10k_Classic](/img/themes/powerlevel10k_classic.png)][Powerlevel10k_Classic]
+
+### [Powerlevel10k_Lean]
+
+[![Powerlevel10k_Lean](/img/themes/powerlevel10k_lean.png)][Powerlevel10k_Lean]
+
+### [PowerLine]
+
+[![PowerLine](/img/themes/powerline.png)][PowerLine]
+
+### [RobbyRussel]
+
+[![RobbyRussel](/img/themes/robbyrussel.png)][RobbyRussel]
+
+### [Sorin]
+
+[![Sorin](/img/themes/sorin.png)][Sorin]
+
+### [Star]
+
+[![Star](/img/themes/star.png)][Star]
+
+### [Zash]
+
+[![Zash](/img/themes/zash.png)][Zash]
+
+[Agnoster]: https://github.com/JanDeDobbeleer/oh-my-posh3/blob/main/themes/agnoster.json 'Agnoster'
```

Comment:
we need to rename all the theme files to `*.omp.json`, I just changed that 🤦🏻

Reply:
@CatTheHacker all that's needed is this change and we're good to go!

Reply:
Fixed.